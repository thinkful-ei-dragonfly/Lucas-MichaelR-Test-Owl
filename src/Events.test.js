import React from 'react';
import renderer from 'react-test-renderer';

import Event from './Events';

const event = {
  participantId: 3,
  type: 'leave-stage',
  timestamp: 1548852724247
};

describe('<Events />', () => {
  it('renders without crashing [snapshot test]', () => {
    // Render this component as JSON
    const tree = renderer.create(<Event key={1} userId={event.participantId} type={event.type} time={event.time} message={event.message} timestamp={event.timestamp} />).toJSON();
    // Checks wether this matches the previous snapshot
    // The previous snapshot will be stored in __snapshots__/Events .test.js
    expect(tree).toMatchSnapshot();
  })
})