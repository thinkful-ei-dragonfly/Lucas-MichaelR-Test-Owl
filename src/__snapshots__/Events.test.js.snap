// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Events /> renders without crashing [snapshot test] 1`] = `
<p>
  <span
    className="chat-user-name"
  >
    Axe Kubicka
  </span>
   joined
</p>
`;
